components/navigation/01-navigation/readme.md

```diff
@@ -0,0 +1,13 @@
+Main site Navigation, containing [Menu](/components/detail/menu) and [Secondary Download Button](/components/detail/download-button--secondary) components.
+
+- On wide viewports, navigation content is displayed inline.
+- On small viewports, navigation content is hidden by default and toggled via a menu icon.
+    - Add `has-label` to the button to display the text label with the menu icon.
+- The logo is always visible.
+    - Switch to the Firefox logo by adding the class `mzp-t-firefox` above the element with `class="mzp-c-navigation-logo"`
```

Comment:
changed from "on" to "above" due to use of [parent selector in scss](https://github.com/mozilla/protocol/blob/0ea311c61f8773cabd5d45409c62594321bd0ff9/src/assets/sass/protocol/components/_navigation.scss#L114)